AppCenterDistribute/AppCenterDistribute/MSACDistributeDelegate.h

```diff
@@ -43,6 +45,13 @@ NS_SWIFT_NAME(DistributeDelegate)
  */
 - (void)distributeWillExitApp:(MSACDistribute *)distribute;
 
+/**
+ * Callback method to inform an ASWebAuthenticationSession from which window it should present content to the user.
+ *
+ * @param distribute The instance of MSACDistribute.
+ */
+- (ASPresentationAnchor)distributeAuthenticationPresentationAnchor:(MSACDistribute *)distribute API_AVAILABLE(ios(13));
```

Comment:
This probably can be removed to reduce the surface area of the framework.

Reply:
@lickel What cases require implementing this callback? Trying to figure out if we can avoid implementing new public API

Reply:
If you’re using multi window mode, you might want to customize the target window. 

It’s an admittedly rare situation. I don’t need it myself—the default implementation was fine.